### ISSUE TITLE
Crash when remote piloting Jade Bot as Core class

### ISSUE THREAD

**Issue by Dave247** [Maselkov/GW2RPC#35]
The app crashes when you remotely pilot a jade bot, such as in Garden Hights in New Kaineng City, which is an activity for the heart there.

```
2023-12-23 19:32:23,721:CRITICAL:root: GW2RPC has crashed
Traceback (most recent call last):
  File "gw2rpc\gw2rpc.py", line 724, in main_loop
  File "gw2rpc\gw2rpc.py", line 448, in get_activity
  File "gw2rpc\character.py", line 58, in __init__
  File "gw2rpc\character.py", line 66, in get_elite_spec
KeyError: 10
```

**Comment by n1tr0-5urf3r** [comment]
Hi there! Are you on the latest version? Jade bots are handled and should not make the game crash. Did you do anything else beside entering a jade bot terminal? 
I wonder what profession ID 10 represents, as there are only 9 professions in GW2. What class and elite spec were you playing?

I will add some error handling to prevent crashes like that though.

![grafik](https://github.com/Maselkov/GW2RPC/assets/10401990/6320a0d5-41ec-4cd7-92ce-7ea739dd2cb4)

**Comment by Dave247** [comment]
Yeah I made sure to download 2.53 from the githib here just in case. The crash happens right after using the terminal. I stopped controlling the bot, started up the app and tried again by accessing the terminal, and it crashed as expected.

I didn't have any elite spec equipped, just running a basic condi engineer build.

**Comment by n1tr0-5urf3r** [comment]
Interesting, if being a core spec, the profession attribute is rewritten to "Jade Bot" and if being an elite spec, the race attribute is rewritten. Should be fixed now in the new v2.54. Can you test it?

**Comment by Dave247** [comment]
Yep, I can confirm [2.54](https://github.com/Maselkov/GW2RPC/releases/tag/v2.54) fixes it, and can also confirm that it was only being triggered by a core class. Before updating I tried switching to an elite spec and it worked as it should.
